apps/api/src/purchase-orders/purchase-orders.controller.ts

```diff
@@ -34,6 +40,7 @@ export class PurchaseOrdersController {
   @Post()
   @HttpCode(HttpStatus.CREATED)
   @UsePipes(new ValidationPipe({transform: true}))
+  @Permission('create', 'purchase_order')
```

Comment:
Example of only allowing this endpoint for users with the "create" purchase order permission.